apps/oauth2/src/components/OAuthItem.vue

```diff
@@ -28,7 +28,7 @@
 			<div class="action-secret">
 				<code>{{ renderedSecret }}</code>
 				<NcButton type="tertiary-no-background"
-					:aria-label="t('oauth2', 'Show client secret')"
+					:aria-label="t('oauth2', `${renderSecret ? 'Hide client secret' : 'Show client secret' }`)"
```

Comment:
The value of the `t` function should be a constant string so the translation key can be extract

```suggestion
					:aria-label="renderSecret ? t('oauth2', 'Hide client secret') : t('oauth2', 'Show client secret')"
```

In this case, I'd move it to a computed to not overload the template.